### ISSUE TITLE
Findings after first deploy

### ISSUE THREAD

**Issue by sergei-maertens** [maykinmedia/open-personen#140]
I'm deploying Open Personen on a K8S cluster with a generic Django app role, and I noticed some assumptions in that role that should probably be fixed in Open Personen.

1. there is no homepage. The healh check is configured to hit `/` as path, but that 404's. I've updated my role to do the healthcheck against `/api` instead, but a sane homepage as entry-point should be added imo.

2. `openpersonen.conf.docker` settings are not maximum secure. I had to override the following undocumented defaults:
    
    - `SESSION_COOKIE_SECURE`
    - `SESSION_COOKIE_HTTPONLY`
    - `CSRF_COOKIE_SECURE`

    Basically, the `docker` settings module is too much geared towards development. Please take a look at the Open Zaak settings organization where everything is available in a secure manner from `base.py`. In the same vein, the included `production` settings module contains a secret key, which just looks bad (I am aware that it's not an actual risk right now).

3. the documentation does not hint at the need for persistent volumes/storage for `MEDIA_ROOT`

4. I've got a security issue that I'm taking up in private

**Comment by joeribekker** [comment]
@joeribekker will pick up 1

**Comment by shea-maykinmedia** [comment]
All points have now been addressed.
